<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShareDialog</class>
 <widget class="QDialog" name="ShareDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>385</width>
    <height>393</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Share with Friends</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>3</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,0,0,0">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Choose something to share:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="track">
       <property name="text">
        <string>Track</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="artist">
       <property name="text">
        <string>Artist</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="album">
       <property name="text">
        <string>Album</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1">
      <item>
       <widget class="HttpImageWidget" name="icon">
        <property name="text">
         <string>icon</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="shareSubject">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="shareSubject2">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="with">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>With:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="ItemSelectorWidget" name="recipients">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="messageTitle">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Message (optional):</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="message">
     <property name="tabChangesFocus">
      <bool>true</bool>
     </property>
     <property name="acceptRichText">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0">
     <item>
      <widget class="QCheckBox" name="isPublic">
       <property name="text">
        <string>include in my recent activity</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="characterLimit">
       <property name="text">
        <string>0/1000</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttons">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ItemSelectorWidget</class>
   <extends>QFrame</extends>
   <header>lib/unicorn/widgets/ItemSelectorWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>HttpImageWidget</class>
   <extends>QLabel</extends>
   <header>lib/unicorn/widgets/HttpImageWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttons</sender>
   <signal>accepted()</signal>
   <receiver>ShareDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>81</x>
     <y>280</y>
    </hint>
    <hint type="destinationlabel">
     <x>4</x>
     <y>234</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttons</sender>
   <signal>rejected()</signal>
   <receiver>ShareDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>113</x>
     <y>272</y>
    </hint>
    <hint type="destinationlabel">
     <x>-7</x>
     <y>271</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
